app/v1/dist/index.html

```diff
@@ -187,6 +193,14 @@
                                                       .onChange((value) => { dataset.mesh.visible = value; });
                             }
                           }
+                          
+                          if(file.translator){
```

Comment:
Let's be boring and put spaces after if

if (translator) {
  ...
} else {
  ...
}